cmake/usCTestScript_custom.cmake

```diff
@@ -23,7 +23,7 @@ set(US_TEST_STATIC 1)
 set(US_SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/../")
 
 set(US_BUILD_CONFIGURATION )
-foreach(i RANGE 7)
+foreach(i RANGE 3)
```

Comment:
For me, it would still be okay like this, if there is no easy solution to it. I hope that the number of configurations does not increase any more.